tests/unit_tests/test_wave_03.py

```diff
@@ -131,7 +131,8 @@ def test_swap_items_from_their_empty_returns_false():
 
     result = fatimah.swap_items(jolie, item_b, nobodys_item)
 
-    raise Exception("Complete this test according to comments below.")
+    assert result == False
+    assert not nobodys_item in fatimah.inventory
```

Comment:
The slightly more pythonic way would be to place "not" in front of "in" so we have "not in fatimah.inventory"